src/components/custom-field-input-date/custom-field-input-date.test.jsx

```diff
@@ -45,25 +45,32 @@ describe('src/components/custom-field-input-date/custom-field-input-date', () =>
     it('has a name attribute; returns name prop', () => {
       const ref = createRef();
       render(<CustomFieldInputDate {...requiredProps} ref={ref} />);
-      expect(ref.current.name).toEqual('field-id');
+      expect(ref.current.name).toEqual('field-name');
     });
 
     it('has a value attribute; returns the value in yyyy-mm-dd format', () => {
       const ref = createRef();
-      render(<CustomFieldInputDate {...requiredProps} ref={ref} value={'09/27/2020'} />);
+      render(<CustomFieldInputDate {...requiredProps} ref={ref} value="2020-09-27" />);
       expect(ref.current.value).toEqual('2020-09-27');
     });
   });
 
+  describe('className API', () => {
+    it('sets the input class name', () => {
+      render(<CustomFieldInputDate {...requiredProps} className="unique-input" />);
+      expect(document.body).toMatchSnapshot();
+    });
```

Comment:
Is it worth writing an expectation that shows that the class name is respected? I imagine it's on the snapshot but that's not immediately obvious from the spec

Reply:
Yeah, it's in the snapshot. Whenever the DOM changes, a developer will need to regenerate the snapshot and make a conscious decision to accept the new markup. For this test, they will focus on class names on the input. 

I still haven't found the perfect balance between snapshots vs exact assertions. 

In the `date/date.test.jsx` suite, we have a similar test for `classNames API`. It makes more sense to use a snapshot on that since it tests 3 difference class names. I think repeating the pattern for `className API` is probably going to be a lower mental overhead.